ar_exercise/exercise.rb

```diff
@@ -34,16 +34,23 @@ class Apartment < ActiveRecord::Base
 all_tenants = Tenant.all
 
 # get the first tenant in the DB
+first_tenant = Tenant.find(1)
 # get all tenants older than 65
+ova_65 = Tenant.where('age > 65')
+
 # get all apartments whose price is greater than $2300
+hi_price = Apartment.where('monthly_rent > 2300')
 # get the apartment with the address "6005 Damien Corners"
+spec_address = Apartment.where('address == "6005 Damien Corners"')
 # get all tenants in that apartment
 
 # Use `each` and `puts` to:
 # Display the name and ID # of every tenant
+name_addres = Tenant.all.each(|ten| p :name, :id)
 # Iterate over each apartment, for each apartment, display it's address and rent price
+add_price = Apartment.all.each{|ten| puts :address, :monthly_rent}
 # Iterate over each apartment, for each apartment, display it's address and all of it's tenants
-
+ap_add_ten = Apartment.all.each.map{|ten| puts :address, (Tenants.all.each{puts :}) }
```

Comment:
This looks like it'll error out. Let me know if you have any questions about how to write this one out.